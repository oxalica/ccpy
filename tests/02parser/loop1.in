for x in xs:
  for a, b in (1, 2), (3, 4):
    pass

while 1 + 2:
  3 + 4
