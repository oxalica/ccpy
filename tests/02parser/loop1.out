StmtFor { PatName("x"), ExprName("xs"), [StmtFor { PatTuple[PatName("a"), PatName("b")], ExprTuple(ExprTuple(ExprLiteral(LitInteger(1)), ExprLiteral(LitInteger(2))), ExprTuple(ExprLiteral(LitInteger(3)), ExprLiteral(LitInteger(4)))), [StmtPass] }] }
StmtWhile { ExprBinary("add", ExprLiteral(LitInteger(1)), ExprLiteral(LitInteger(2))), [StmtExpr(ExprBinary("add", ExprLiteral(LitInteger(3)), ExprLiteral(LitInteger(4))))] }
End
